x-pack/plugins/observability_solution/inventory/public/components/entities_grid/index.tsx

```diff
@@ -180,6 +183,24 @@ export function EntitiesGrid({
     [assetDetailsLocator, serviceOverviewLocator]
   );
 
+  const renderNameCellValue = useCallback(
+    (entity: LatestEntity) => {
+      return (
+        <EuiLink data-test-subj="inventoryCellValueLink" href={getEntityRedirectUrl(entity)}>
```

Comment:
Can you please create a separate component for entity display column  to follow the single responsibility principle? This will make the component easier to test and we can reuse it.